go-controller/pkg/util/nicstobridge_test.go

```diff
@@ -0,0 +1,1360 @@
+			},
+		},
+		{
+			desc:      "test code path when input bridge name has prefix `br` and stdout is empty",
+			outputExp: "Name",
+			inpBrName: "brName",
+			onRetArgsExecIface: []onCallReturnArgs{
+				{"RunCmd", []string{"*mocks.Cmd", "string", "[]string", "string", "string", "string", "string"}, []interface{}{bytes.NewBuffer([]byte("port1")), bytes.NewBuffer([]byte("")), nil}},
+				{"RunCmd", []string{"*mocks.Cmd", "string", "[]string", "string", "string", "string", "string", "string"}, []interface{}{bytes.NewBuffer([]byte("[080841d8-4e12-4003-b0ac-36fefd63bae1]")), bytes.NewBuffer([]byte("")), nil}},
+				{"RunCmd", []string{"*mocks.Cmd", "string", "[]string", "string", "string", "string", "string", "string"}, []interface{}{bytes.NewBuffer([]byte("internal")), bytes.NewBuffer([]byte("")), nil}},
+				{"RunCmd", []string{"*mocks.Cmd", "string", "[]string", "string", "string", "string", "string"}, []interface{}{bytes.NewBuffer([]byte("")), bytes.NewBuffer([]byte("")), nil}},
+			},
+			onRetArgsKexecIface: []onCallReturnArgs{
+				{"Command", []string{"string", "string", "string", "string", "string"}, []interface{}{mockCmd}},
+				{"Command", []string{"string", "string", "string", "string", "string", "string"}, []interface{}{mockCmd}},
+				{"Command", []string{"string", "string", "string", "string", "string", "string"}, []interface{}{mockCmd}},
+				{"Command", []string{"string", "string", "string", "string", "string"}, []interface{}{mockCmd}},
+			},
+		},
+		{
+			desc:      "test code path when input bridge name has NO prefix `br`",
+			outputExp: "someoutput",
+			inpBrName: "testName",
+			onRetArgsExecIface: []onCallReturnArgs{
+				{"RunCmd", []string{"*mocks.Cmd", "string", "[]string", "string", "string", "string", "string"}, []interface{}{bytes.NewBuffer([]byte("port1")), bytes.NewBuffer([]byte("")), nil}},
+				{"RunCmd", []string{"*mocks.Cmd", "string", "[]string", "string", "string", "string", "string", "string"}, []interface{}{bytes.NewBuffer([]byte("[080841d8-4e12-4003-b0ac-36fefd63bae1]")), bytes.NewBuffer([]byte("")), nil}},
+				{"RunCmd", []string{"*mocks.Cmd", "string", "[]string", "string", "string", "string", "string", "string"}, []interface{}{bytes.NewBuffer([]byte("internal")), bytes.NewBuffer([]byte("")), nil}},
+				{"RunCmd", []string{"*mocks.Cmd", "string", "[]string", "string", "string", "string", "string"}, []interface{}{bytes.NewBuffer([]byte("someoutput")), bytes.NewBuffer([]byte("")), nil}},
+			},
+			onRetArgsKexecIface: []onCallReturnArgs{
+				{"Command", []string{"string", "string", "string", "string", "string"}, []interface{}{mockCmd}},
+				{"Command", []string{"string", "string", "string", "string", "string", "string"}, []interface{}{mockCmd}},
+				{"Command", []string{"string", "string", "string", "string", "string", "string"}, []interface{}{mockCmd}},
+				{"Command", []string{"string", "string", "string", "string", "string"}, []interface{}{mockCmd}},
+			},
+		},
+		/*{
```

Comment:
why is this commented out?

Reply:
Oops, I need to remove the commented out tests, I have done a test refactor to accomodate the changes you made in an earlier PR that had changes to nictobridge.go
The commented ones need to be removed and I shall do it shortly and push an update to the PR